scripts/cpplint.py

```diff
@@ -6365,6 +6380,23 @@ def ProcessFileData(filename, file_extension, lines, error,
 
   ResetNolintSuppressions()
 
+  # Load module dependencies
+  module_deps_file = os.path.join(os.path.dirname(filename), 'module_dependencies.txt')
```

Comment:
For `src/ansi-c/literals/convert_float_literal.cpp` this will go looking in `src/ansi-c/literalsl/module_dependencies.txt` no? Which presumably isn't quite right? I guess we need to find the last `src/` and then the folder immediately below that?

Reply:
My mistake - I see sub folders get their own dependencies.